src/overlays/actors/ovl_Bg_Ganon_Otyuka/z_bg_ganon_otyuka.c

```diff
@@ -31,18 +32,360 @@ const ActorInit Bg_Ganon_Otyuka_InitVars = {
     (ActorFunc)BgGanonOtyuka_Update,
     (ActorFunc)BgGanonOtyuka_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Ganon_Otyuka/BgGanonOtyuka_Init.s")
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 1000, ICHAIN_STOP),
+};
+
+static u8 sSides[] = { SIDE_EAST, SIDE_WEST, SIDE_SOUTH, SIDE_NORTH };
+
+static Vec3f D_80876A68[] = {
+    { 120.0f, 0.0f, 0.0f },
+    { -120.0f, 0.0f, 0.0f },
+    { 0.0f, 0.0f, 120.0f },
+    { 0.0f, 0.0f, -120.0f },
+};
+
+static Color_RGBA8 sDustPrimColor = { 60, 60, 0, 0 };
+
+static Color_RGBA8 sDustEnvColor = { 50, 20, 0, 0 };
+
+static Vec3f sSideCenters[] = {
+    { 60.0f, 0.0f, 0.0f },
+    { -60.0f, 0.0f, 0.0f },
+    { 0.0f, 0.0f, 60.0f },
+    { 0.0f, 0.0f, -60.0f },
+};
+
+static f32 sSideAngles[] = { M_PI / 2, -M_PI / 2, 0.0f, M_PI };
+
+#include "z_bg_ganon_otyuka_gfx.c"
+
+static CamData sCameraDataList[] = { { 0, 0, 0 } };
+
+static UNK_TYPE sSurfaceTypeList[] = {
+    0x00000000,
+    0x000007C0,
+    0x00000000,
+    0x000007C2,
+};
+
+static CollisionPoly sPolyList[] = {
+    { { 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x00, 0x02 }, { 32767, 0, 0 }, -60 },
```

Comment:
Writing data to a filler (`char unk_00[0x8]`) is a bit awkward, but I guess we don't know what type these are just yet? (Mostly a comment for other people looking at this PR)

Reply:
I'm pretty sure this data structure has a proper layout in bgcheck, but that hasn't been merged yet.

Reply:
id bring in the relevant struct from the bg check pr in this case. if nothing else is using it it shouldnt cause any conflicts

Reply:
I believe a lot of things were changed in the struct (including the name?), so that's probably not a good idea.